### PR TITLE
Add sandbox attribute to omid_v1_present iframe

### LINKED ISSUE #30: omid_v1_present iframe missing sandbox attribute
# Overview
The iframe created to detect if omid is present, or not, does not contain a `sandbox` attribute. 

Current output:
```html
<iframe id="omid_v1_present" name="omid_v1_present" style="display: none;">
...
</iframe>
```

Desired output:
```html
<iframe id="omid_v1_present" name="omid_v1_present" style="display: none;" sandbox>
...
</iframe>
```

[Code reference](https://github.com/InteractiveAdvertisingBureau/Open-Measurement-JSClients/blob/9a7ff8be0c0a9e196bddbd4b0642583b910b7cfd/src/common/detect-omid.js#L54).

## Why is this a problem? 
This is a problem because the `sandbox` attribute "controls the restrictions applied to the content embedded in the <iframe>" ([ref](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#sandbox)). Although this iframe only exists so that verfication scripts can find where the OMSDK is, it is best practice to enable all iframe restrictions.

## What can be done?
Add `sandbox` to the iframe.

### PR DESCRIPTION
**Pull request by bjoberg**
Resolves #30

### CODE REVIEW

**Comment by nlehrer** [comment]
Thanks, Brett. What we will do is merge this into the upstream repo and then it'll end up here after the next release. I'll let you know once it is merged and then we can close this PR. If you want to contribute more to the upstream repo you can join the commit group (email [omsdksupport@iabtechlab.com](mailto:omsdksupport@iabtechlab.com) for more info).

**Comment by bjoberg** [comment]
Sounds good @nlehrer, thanks for clarifying. Is there a release schedule I can follow?

**Comment by nlehrer** [comment]
There is a release about once every two weeks. We haven't published an official release schedule, but I can ask about that possibility. We're planning to do a release sometime this week.

**Comment by bjoberg** [comment]
@nlehrer has then been merged into the upstream repo and should I close this PR?

File: src/common/detect-omid.js [hunk 1]
```diff
@@ -66,6 +67,7 @@ exports.appendPresenceIframe_ = function(globalObject) {
   iframe.id = exports.OMID_PRESENT_FRAME_NAME;
   iframe.name = exports.OMID_PRESENT_FRAME_NAME;
   iframe.style.display = 'none';
+  iframe.sandbox = undefined;
```
Comment (nlehrer) [inline src/common/detect-omid.js:70]: FYI setting this to undefined causes the browser to complain; setting it to empty string seems to be the right way to set it to just "sandbox".

**Comment by nlehrer** [comment]
It's going to be merged today or tomorrow. I'm closing this one meanwhile. Thanks!

File: src/common/detect-omid.js (hunk 1 above)
Comment (bjoberg) [reply src/common/detect-omid.js:70]: Oh interesting... Thanks for the follow up. Want me to make this change, or will you do it in the other repo?

Comment (nlehrer) [reply src/common/detect-omid.js:70]: I did it in the other repo PR. Thanks.

**Pull request closed without merging**